### PR TITLE
feat: copy tooltip

### PR DESCRIPTION
**Pull request by shrugs**
took the liberty of grabbing an open source icon and a simple tooltip library

### CODE REVIEW

**Comment by shrugs** [comment]
huh, github literally shipped this feature since we've been talking about it

copy

**Comment by shrugs** [comment]
hold up that package-lock addition is wild.

**Comment by shrugs** [comment]
what the fuck?

**Comment by shrugs** [comment]
what the absolute fuck is this `synk` shit? it's got so many fucking dependencies

**Comment by shrugs** [comment]
I'm ripping this library out and replacing it

**Comment by shrugs** [comment]
Now That's What I Call Open Source™

**Comment by shrugs** [comment]
copy

**Comment by shrugs** [comment]
mine even looks better. ¯\\\_(ツ)_/¯

File: src/components/CopiableWithTooltip.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import WithTooltip from 'components/WithTooltip';
+
+export default function CopiableWithTooltip({
+  as: As = 'span',
+  text,
+  children,
+  className,
+  ...rest
+}) {
+  const [doCopy, didCopy] = useCopiable(text || children);
```
Comment (Fang-) [inline src/components/CopiableWithTooltip.js]: When, if ever, does `didCopy` get cleared?

Comment (shrugs) [reply src/components/CopiableWithTooltip.js]: it gets cleared after 2 seconds within the `useCopiable` logic

**Comment by shrugs** [comment]
@Fang- yeah I couldn't figure out how to get the icon to actually vertically align correctly, without changing the container to flex (which might have interfered with the layout everywhere else in the app?). after like 10 minutes of trying I gave up and figured it wasn't worth it lol

**Pull request merged**